lib/core/src/lib/viewer/components/viewer.component.ts

```diff
@@ -344,6 +344,10 @@ export class ViewerComponent<T> implements OnDestroy, OnInit, OnChanges {
         this.submitFile.emit(newImageBlob);
     }
 
+    onSavingFile(isSaving: boolean) {
```

Comment:
Also here you can do that assigning in html directly probably as that function has nothing more, like: 
(isSaving)="allowNavigate = !$event"